src/lib/server/database/models/index.ts

```diff
@@ -51,6 +51,9 @@ export const confession = app.table(
         approvedAt: timestamp('approved_at', { withTimezone: true }).defaultNow(),
         authorId: bigint('author_id', { mode: 'bigint' }).notNull(),
         content: text('content').notNull(),
+        attachmentUrl: text('attachment_url'),
+        attachmentFilename: text('attachment_filename'),
+        attachmentType: text('attachment_type'),
```

Comment:
I would prefer if we have a dedicated `attachment` table for this so that the fields are `NOT NULL` while the foreign key may be nullable.

And on that note, we should leverage the [Relations API](https://orm.drizzle.team/docs/relations) to make querying for attachments convenient.

Reply:
Refer to the conversation in `/database/index.ts`